### ISSUE TITLE
Add night mode

### ISSUE THREAD

**Issue by SergeyZhukovsky** [brave/browser-android-tabs#81]

**Comment by LaurenWags** [comment]
+1 Google play store

**Comment by Dudes1** [comment]
One up.

**Comment by yodrew** [comment]
Yes! Brave is great, but it is blinding me.

**Comment by LaurenWags** [comment]
+1 Google Play Store

**Comment by SergeyDoc** [comment]
I would love to see not just a night mode, but a completely black background and light letters (i.e.text) on it. It's useful to Amoled displays, helps to reduce battery consumption and diminishes the eye strain. 
The only Browser I know where it is possible is UC Browser. I made it black by applying simultaneously a completely black theme and a transparent background. It would be lovely to have something like that for Brave.

**Comment by brotherfunk** [comment]
Please

**Comment by pdimri** [comment]
Anytime line for adding dark/night mode feature ?
I am really looking for this feature in brave like other caf based browsers have.

**Comment by Hamburger6** [comment]
Looking for a dark mode too. Like the browser and it's AD blocking ability.

**Comment by eljuno** [comment]
+1 from Twitter https://twitter.com/MOregonRA/status/1010394888322469888

**Comment by eljuno** [comment]
+1 from Twitter https://twitter.com/_ROB_29/status/1012088249987133442

**Comment by eljuno** [comment]
+1  from Twitter https://twitter.com/Food4ears/status/1013009287105449986

**Comment by Hal47** [comment]
The lack of a dark theme/night mode is pretty much the only reason I haven't switched to Brave yet.

**Comment by Hamburger6** [comment]
I'm always using brave as well. It's My default browser and My only browser on My phone. Battery consumption is somewhat fast so a dark or night mode would help.

**Comment by Pkshields** [comment]
+1

With the introduction of the ever-brighter white theme in Chrome 69, a darker theme in Brave Android is definitely going to be needed soon.

**Comment by Hamburger6** [comment]
Very true.

**Comment by tobia** [comment]
When I really need dark mode (for example, if browsing in the middle of the night while needing to maintain night vision, or browsing in a place where a full lit screen is troublesome for people around you, even at minimum brightness) my options on Android are currently: 

1. Use Firefox's reader mode and choose the dark theme (which is not even fully white on black, shame!)
2. Use Yandex browser with the Deluminate extension
3. Use my phone's (OnePlus) option to invert the whole screen

Yandex is the only Android browser I could find that supports Chrome extensions, and Deluminate is by far the best Dark Mode for websites ever developed for any platform.

Needless to say, both options are sub-optimal and I'd love for someone to take Deluminate's fine-tuned algorithm (and site-specific settings panel) and port them to Brave.

**Comment by thanuj10** [comment]
+1 for this

**Comment by Hamburger6** [comment]
When can we get a dark or night mode on brave browser? Still waiting.

**Comment by Brave-Matt** [comment]
+1 from Reddit: 
https://www.reddit.com/r/brave_browser/comments/9mp126/dark_theme_android/

**Comment by michaelardo** [comment]
+1 from me.
Would be great to see it implentented

**Comment by tsumeone** [comment]
+1.  I won't use a browser on my phone unless it has dark mode, which is a shame because Brave has everything else.  For now I use Edge.  Firefox and derivatives perform slowly on mobile, and Samsung browser turns the whole page dark (broken implementation as far as I'm concerned).  I'd much rather use Brave.

**Comment by Crimix** [comment]
With the new very light and white design of chrome, I had hoped that this feature was on the way

**Comment by Hamburger6** [comment]
This has been an issue for awhile. Needs dark mode! Following chrome too much.

**Comment by Hamburger6** [comment]
STILL waiting on dark theme for Brave browser. Using second browser for general use with night mode because of this. Not always as secure, but nice looking.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/dark-theme-and-save-sync-favorites-on-mobile/37034

**Comment by Hamburger6** [comment]
When will Android brave have dark mode then?

**Comment by gcasanache** [comment]
Hello there,

Any news?

**Comment by aaylnx** [comment]
> Hello there,
> 
> Any news?

If only!

**Comment by srirambv** [comment]
Its on the list but not priority. We are working on other major features like sync and rewards. If you have a fix for this please open a PR and one of the dev would review it and can be pushed as soon as possible.

**Comment by gcasanache** [comment]
Makes sense. Keep up the good work.

**Comment by kubapolish** [comment]
+1, it has been 2 years, please add this

**Comment by thepinkturtle** [comment]
Come on Devs. This can't be that hard. Even the Samsung "Internet" android app has a dark theme, Reddit app has a dark theme, Twitter app has a dark theme, Google maps switches over to dark theme at night and there are plenty more examples. Just make it a beta option and let us switch it on and off and then the community will provide feedback. At the very least, update everyone on the current situation i.e., an extension store will be opened with the ability to apply custom themes.

**Comment by aaylnx** [comment]
> Come on Devs. This can't be that hard. Even the Samsung "Internet" android app has a dark theme, Reddit app has a dark theme, Twitter app has a dark theme, Google maps switches over to dark theme at night and there are plenty more examples. Just make it a beta option and let us switch it on and off and then the community will provide feedback. At the very least, update everyone on the current situation i.e., an extension store will be opened with the ability to apply custom themes.

I agree.  Lots of people are clamoring for this.  Please prioritize.

**Comment by Brave-Matt** [comment]
+1 from Reddit:
https://www.reddit.com/r/brave_browser/comments/a5halp/dark_mode/

**Comment by aaylnx** [comment]
> +1 from Reddit:
> https://www.reddit.com/r/brave_browser/comments/a5halp/dark_mode/

Thanks for the link.  

Nice to see that night/dark mode is looking more like a possibility.  The Reddit post indicates that it's already in a dev version of the desktop version of Brave.  Hopefully, this means Android will get it too.

**Comment by stevepereira** [comment]
+1 From a new user and committed chrome switcher

**Comment by gcasanache** [comment]
Bump.

**Comment by srirambv** [comment]
+1 from https://community.brave.com/t/android-update-when-it-may-come/43441

**Comment by holyavengerone** [comment]
+1 from reddit. Go go dark mode!

**Comment by nivinj** [comment]
+1 - need this for Android. Desktop dark theme is looking great.

**Comment by Dho100** [comment]
+1 this is a high priority for me. More so than the mentioned sync features currently being slated for development next. Thank you!

**Comment by iButane** [comment]
+1 for dark/night mode

**Comment by Hamburger6** [comment]
Seriously. Very blinding.

**Comment by jonathansampson** [comment]
Recently requested on Twitter: https://twitter.com/Joeligj12/status/1101350717518987264?s=19

**Comment by theSurgeonoftheNightSky** [comment]
Yes, please.

**Comment by Hamburger6** [comment]
NEED NIGHT MODE!

**Comment by pdimri** [comment]
They can't provide. The are not capable of.

**Comment by aaylnx** [comment]
After having discovered how much battery life it saves, google is making a
big push to make Android much more night mode friendly and this includes
Chrome.  See the following article.

https://www.androidpolice.com/2019/02/27/you-can-now-enable-dark-mode-for-web-pages-and-content-in-chrome-canary/

In so far that Brave can take this from upstream Chrome/Chromium, maybe it
will happen in the not so distant future.

**Comment by Hamburger6** [comment]
Someone says that brave browser can't do a night mode? Why?

**Comment by pdimri** [comment]
Because Night mode had been requested long long time back and brave team
unitl now could not deliver that. I doubt their capability, meantime other
browsers implemented this feature.

**Comment by jonathansampson** [comment]
@pdimri Please understand that this has nothing to do with ability, but rather capacity. Our engineers are presently working very hard to deliver other features in the Android build, including Sync and Rewards. We love the idea of Dark Mode, but will need to focus on that a bit later. Please be patient; our goal is to build the most feature-rich and enjoyable experience possible.

**Comment by Dho100** [comment]
I'm not interested in sync, or rewards. Highest priority feature and experience for me is night mode.

**Comment by Quazo** [comment]
+1

**Comment by Navds** [comment]
It reminds me of Microsoft's obsession with useless Skype updates, ignoring users call for  nothing but a simple yet working app. Sorry, this is all but usable. These people don't look for aesthetic, they want a browser for intense use.

**Comment by lhorace** [comment]
> I'm not interested in sync

So the developers should drop this and focus on `Night Mode`? Watch https://youtu.be/dckiJ-vycpY?t=40 to see why sometimes Dark Mode in browsers could be useless. Alternatively, if Brave support Theme extensioins, e.g Firefox which has a Dark Mode theme, be a better idea?

> These people don't look for aesthetic, they want a browser for intense use.

I want aesthetic but wouldn't `Theme extensions` be a better avenue, if you agree with, than commit developer resources on this?

**Comment by fhelwanger** [comment]
Yeah most of time when people complaining say "ignoring users" then really meant "ignoring me" 😆 

Although I'd really like dark mode (that's why I'm here), the devs are already aware of it but they're prioritizing other features (as already previously said). They have more data than us to say which feature is more important.

Instead of giving useless complains and comments here, which trigger a lot of useless notifications for a lot of people that want news on dark mode, just give a 👍 to the issue (which is the most efficient way to say "I want this") and move on. Eventually it'll be implemented.

**Comment by Dho100** [comment]
"My eye sight was getting worse everytime I had to hardcore-study for exams (12h/day for min. 2 months). I read that scientists found out that black text on white ground was causing your eyeballs to grow and develop shortsightedness, while white text on black ground stops this process."

https://www.reddit.com/r/YouShouldKnow/comments/b3ul4v/ysk_about_dark_reader_a_browser_addon_that/

**Comment by CryptoMiike** [comment]
If developers will continue to ignore our ideas and take it easy I don't see this browser be TOP10 any time soon. Recently I have been testing a few major browsers and they all have a dark mode. It is a new standard now. If you don't have it don't be surprised why your user grow index doesn't increase as you want. No dark mode included no Brave as a default browser.

**Comment by thepinkturtle** [comment]
What other mobile browsers have you been using, I just curious?

- Regards Devin -

________________________________

**Comment by HennyBro** [comment]
Seriously, @bravedevs: You could just say that Dark Mode will come later, definitely and that you are only focusing Brave Rewards and Sync right now. Just confirm a website wide dark mode NOW and we'll have something that we can look forward to...

Please 😊

**Comment by erghbn** [comment]
Please! It's blinding.

**Comment by Alex15Git** [comment]
Night mode please.

**Comment by robert-clayton** [comment]
+1 for Night Mode!

**Comment by blackparallax** [comment]
Definitely need dark/night mode please.

**Comment by shoulderkolibri** [comment]
+1 please, save my precious eyes before they pop like frog eggs bursting in spring ( ◞･౪･)

**Comment by Snickers775** [comment]
I know you guys realize how important a "Dark Mode" is for many of us mobile users. Please make it a priority on your end also. After the sun sets, I have to switch back to Samsung Internet because Brave hurts my eyes!

**Comment by pdimri** [comment]
You can switch to kiwi browser. No point of begging them. I have abandoned
this browser long time.

**Comment by alexwykoff** [comment]
Another request from the public:
https://twitter.com/adalberto_ss/status/1119266279586762752

**Comment by sggts04** [comment]
+1

**Comment by Rnksts** [comment]
+1 — Only thing preventing me from switching.

**Comment by Dho100** [comment]
@jonathansampson is there an ETA for this feature to be released? Thanks

**Comment by SmitTheTux** [comment]
All those features like sync or rewards would be useless if people don't use this browser in the first place cause its bleeding there eyes.

**Comment by avently** [comment]
Chromium already has dark mode (could be enabled via chrome://flags ). So this mode will be here too after some time

**Comment by shoulderkolibri** [comment]
Brave's Dark Mode is in the settings area with a drop-down menu under Appearance ->Brave Colors :D

**Comment by be16242** [comment]
+1

**Comment by goodfore** [comment]
+1

**Comment by dlenkes** [comment]
Can we have the same flags at least as chrome?
https://www.youtube.com/watch?v=qNgGCMR3NzA

**Comment by druby-luke** [comment]
+1 Would love this

**Comment by davidgenesiscruz** [comment]
+1 pleeeeeeeeeaassssse!!!!!!!!!!! 🙏🙏🙏

**Comment by jamesmudgett** [comment]
![Screen Shot 2019-05-16 at 11 04 31 AM](https://user-images.githubusercontent.com/318119/57869311-68fcae00-77ca-11e9-9c28-67c01323b104.png)

PM graphic 
[private graphic.zip](https://github.com/brave/browser-android-tabs/files/3187973/private.graphic.zip)

**Comment by shivangswain** [comment]
+1 from me. Please make this a feature and don't just have a dark mode, add a black (AMOLED) mode as well.

**Comment by mannyfin** [comment]
Here's a workaround for android until it is implemented fully as @avently  hinted at.
1. In Brave mobile, navigate to:  chrome://flags
2. search or find in page "dark"
3. Set the following two features from "default" to "enabled".: a) "Android web contents dark mode" and b) Android Chrome UI dark mode
4. Relaunch mobile brave browser.

The above makes dark mode happen on Android. It also turns the Brave bar dark too!

**Comment by davidgenesiscruz** [comment]
@mannyfin Thanks for the info.
Unfortunately, it just reverses the colors on css for me. The browser interface all remains white.
(And a page that originally has a black background will turn white.)

Brave 1.0.94
Android 9
Samsung S10+

**Comment by mannyfin** [comment]
> 
> 
> @mannyfin Thanks for the info.
> Unfortunately, it just reverses the colors on css for me. The browser interface all remains white.
> (And a page that originally has a black background will turn white.)
> 
> Brave 1.0.94
> Android 9
> Samsung S10+

See my edit above. Good find. I verified it by looking at a [bootstrap dark theme](http://www.bootstrapdash.com/demo/corona/jquery/) I think its due to the "Android web contents dark mode" . When I changed back the setting to the default, the original dark theme was implemented.

**Comment by davidgenesiscruz** [comment]
It worked! Awesome!!

**Comment by walkafwalka** [comment]
Unfortunately, the dark mode is not AMOLED dark mode. However, it looks like we might have to wait for upstream to implement it.

**Comment by wbs-perso** [comment]
+1 for AMOLED "compatible" dark mode

**Comment by akashpjames** [comment]
Is there any update on this?

**Comment by Adrymate9** [comment]
I noticed in the last update 1.4.0 rc2 that the dark mode setting has been manually enabled.  I want to confirm if this feature is only for the Brave interface, because if it includes the view of the web pages, in my case it would not be working.  I await your response, thank you.

**Comment by Hamburger6** [comment]
Do you keep your chrome flags on when using the theme switch on brave.

**Comment by Adrymate9** [comment]
Indeed, one of the 2 flags had not been activated. Thanks for the help. Now another issue is that this latest update causes arrests and breaks when browsing web pages. Strange, until the previous version this did not happen. It becomes quite complicated to navigate fluently.

**Comment by Hamburger6** [comment]
The main issue for me is the reversed color you have sometimes. Hopefully it will be fixed soon.

**Comment by Adrymate9** [comment]
Yes, they still are not entirely correct. I hope they solve it soon. A long time ago users requested the dark mode shortly, it is a breakthrough that is already incorporated.

**Comment by szaimen** [comment]
The funny thing is, that is just coming to Brave-Android because Google implemented this a while ago in Chrome. So they are just using the same feature for Brave-Android.
https://www.xda-developers.com/google-chrome-android-dark-mode/

**Comment by szaimen** [comment]
BTW: with Chrome 78 will be coming the first well working "dark web content" mode to automatically also make all web content dark. Before this version it didn't work good enough. (Inverting images, etc.)
So as soon as Brave is based on Chromium 78, you can enable it via the chrome://flags (or brave://flags) and it will work quiet nicely.

**Comment by Adrymate9** [comment]
in the version I have 1.4.0 rc2 is not yet version 77 of Chrome.  but I do notice an improvement in the signature of displaying the images that do not burn as much as before.  but we need to improve!  esp

 I look forward to version 78, if you say it will be fixed.

**Comment by srirambv** [comment]
> I want to confirm if this feature is only for the Brave interface 

Settings options under appearnce is only on Brave. This is going to be supported on all Android versions. Only Andriod 10 currently supports theme option on existing install.

> you enable it via the chrome://flags  

Flags are experimental so it may not yield exact results. Personally I would suggest enable them only for testing as it may work on some pages and may not on others.
